### PR TITLE
[Term Entry] SwiftUI Views: Circle

### LINKED ISSUE #1092: [Term Entry] SwiftUI Views: Circle
### Reason/inspiration (optional)

Please write a new entry under:
content/swiftui/concepts/views/terms/ about the Circle view in SwiftUI.

A "Syntax" section with pseudo code snippet.
An "Example" section with a swift code snippet and an accompanying image to display the view.

To get started with the PR and entry file:

Under content/swiftui/concepts/views/terms/, create a new folder called circle.
Then create a new file called circle.md under content/swiftui/concepts/views/terms/circle

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by girijakar**
### Description

New term entry for SwiftUI Circle. Added a new folder, new image, and new file for issue #1092. Also, added a new tag in the tags.md
Closes #1092 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/swiftui/concepts/views/terms/circle/circle.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+Title: 'Circle'
+Description: 'Pre-built shape which can be colored, positioned as desired.'
+Subjects:
+  - 'Software Development'
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Circle'
```
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/circle/circle.md:11]: "Circle" may not be a broad enough topic for a tag.

File: content/swiftui/concepts/views/terms/circle/circle.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+SwiftUI includes various pre-built shapes, like rectangles, **circles**, and capsules, which can be generated, coloured, and positioned as desired.
+
+## Syntax
+
+```pseudo
+Circle()
+//Properties, such as fill, frame, borders etc.
+```
+
+The `"Circle"` is one of the pre-built shapes. The user can use different properties such as fill, stroke, alignment, size to adjust the circle according to their requirement.
```
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/circle/circle.md:27]:
```suggestion
The `"Circle"` is one of the pre-built shapes. The user can use different properties such as fill, stroke, alignment, size to adjust the circle according to their requirements.
```

File: documentation/tags.md [hunk 3]
```diff
@@ -46,6 +46,7 @@ Buttons
 Catch
 Characters
 Chatbots
+Circle
```
Comment (SSwiniarski) [inline documentation/tags.md:49]: "Circle" may not be broad enough for a tag.

**Comment by girijakar** [comment]
@SSwiniarski, I implemented your changes.
I have updated the tag from "Circle" to "SwiftUI Circles" in both the circle.md and tags.md. Let me know if this is fine, otherwise, I'll remove this tag.

File: content/swiftui/concepts/views/terms/circle/circle.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+Title: 'Circle'
+Description: 'Pre-built shape which can be colored and positioned as desired.'
+Subjects:
+  - 'Software Development'
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'SwiftUI Circles'
```
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/circle/circle.md:11]:
```suggestion
```

File: documentation/tags.md [hunk 5]
```diff
@@ -256,6 +256,7 @@ Structure
 Style
 SwiftUI
 SwiftUI Views
+SwiftUI Circles
```
Comment (SSwiniarski) [inline documentation/tags.md:259]: I'd get rid of this. The subject is too narrow to be a tag.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@girijakar I had some more comments, basically to remove the SwiftUI Circle tag as it's way too narrow in focus.

**Comment by girijakar** [comment]
@SSwiniarski, removed the tag as suggested.

**Review by SSwiniarski — APPROVED** [review]
@girijakar Looks good for second review

File: content/swiftui/concepts/views/terms/circle/circle.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
+---
+Title: 'Circle'
+Description: 'Pre-built shape which can be colored and positioned as desired.'
```
Comment (yangc95) [inline content/swiftui/concepts/views/terms/circle/circle.md:3]:
```suggestion
Description: 'Pre-built circular shape which can be colored and positioned as desired.'
```

File: content/swiftui/concepts/views/terms/circle/circle.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Views'
+CatalogContent:
+  - 'learn-swiftui'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+SwiftUI includes various pre-built shapes, like rectangles, **circles**, and capsules, which can be generated, coloured, and positioned as desired.
```
Comment (yangc95) [inline content/swiftui/concepts/views/terms/circle/circle.md:17]:
```suggestion
SwiftUI includes various pre-built shapes, like rectangles, **circles**, and capsules which can be generated, coloured, and positioned as desired.
```

File: content/swiftui/concepts/views/terms/circle/circle.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+## Syntax
+
+```pseudo
+Circle()
+//Properties, such as fill, frame, borders etc.
```
Comment (yangc95) [inline content/swiftui/concepts/views/terms/circle/circle.md:23]:
```suggestion
// Modifiers, such as fill, frame, borders etc.
```

File: content/swiftui/concepts/views/terms/circle/circle.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+```
+
+The `"Circle"` is one of the pre-built shapes. The user can use different properties such as fill, stroke, alignment, size to adjust the circle according to their requirements.
```
Comment (yangc95) [inline content/swiftui/concepts/views/terms/circle/circle.md:26]:
```suggestion
`Circle` is one of the pre-built shapes. The user can use different properties such as fill, stroke, alignment, size to adjust the circle according to their requirements.
```

File: content/swiftui/concepts/views/terms/circle/circle.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+
+## Example
+
+The following example make a `"Circle"` which is filled with purple color:
```
Comment (yangc95) [inline content/swiftui/concepts/views/terms/circle/circle.md:30]:
```suggestion
The following example make a `Circle` that is purple:
```

File: content/swiftui/concepts/views/terms/circle/circle.md [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+
+```swift
+  var body: some View{
+    Circle()
+      .fill(.purple)
+      .frame(width: 200; height: 200)
+  }
+```
+
+The above code generates a circle having dimensions of 200x200.
```
Comment (yangc95) [inline content/swiftui/concepts/views/terms/circle/circle.md:40]: Can we explicitly define the measurement unit

File: content/swiftui/concepts/views/terms/circle/circle.md (hunk 6 above)
Comment (yangc95) [reply content/swiftui/concepts/views/terms/circle/circle.md:3]: Instead if saying "colored and positioned" can we say "visually modified?" This makes it broad as to what we can change about the circle view.

**Review by yangc95 — COMMENTED** [review]
@girijakar I think this is very well-written! Please tag me once you take a look over my comments.

Thanks!

File: content/swiftui/concepts/views/terms/circle/circle.md (hunk 11 above)
Comment (girijakar) [reply content/swiftui/concepts/views/terms/circle/circle.md:40]: Yes, I added the unit.

**Comment by girijakar** [comment]
@yangc95, implemented all your changes. Please, have a look and let me know if any more changes are required.

File: content/swiftui/concepts/views/terms/circle/circle.md [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+SwiftUI includes various pre-built shapes, like rectangles, **circles**, and capsules which can be generated, coloured, and positioned as desired.
+
+## Syntax
+
+```pseudo
+Circle()
+// Modifiers, such as fill, frame, and borders
```
Comment (yangc95) [inline content/swiftui/concepts/views/terms/circle/circle.md:23]:
```suggestion
// Modifiers
```
 Let's keep the syntax code snippet like this and explain in the paragraph below

File: content/swiftui/concepts/views/terms/circle/circle.md [hunk 13]
```diff
@@ -0,0 +1,44 @@
...
+```
+
+`Circle` is one of the pre-built shapes. A developer can use different properties such as fill, stroke, alignment, and size to adjust the circle according to their requirements.
```
Comment (yangc95) [inline content/swiftui/concepts/views/terms/circle/circle.md:26]:
```suggestion
`Circle` is one of the pre-built shapes. A developer can use different properties such as fill, stroke, alignment, and size to modify or adjust the circle according to their requirements.
```

File: content/swiftui/concepts/views/terms/circle/circle.md (hunk 12 above)
Comment (yangc95) [reply content/swiftui/concepts/views/terms/circle/circle.md:23]: It'd also be great if we can include that key term below

File: content/swiftui/concepts/views/terms/circle/circle.md [hunk 14]
```diff
@@ -0,0 +1,44 @@
...
+
+## Example
+
+The following example make a `Circle` that is purple:
+
+```swift
+  var body: some View{
+    Circle()
+      .fill(.purple)
+      .frame(width: 200; height: 200)
```
Comment (yangc95) [inline content/swiftui/concepts/views/terms/circle/circle.md:36]:
```suggestion
    .fill(.purple)
    .frame(width: 200; height: 200)
```

File: content/swiftui/concepts/views/terms/circle/circle.md [hunk 15]
```diff
@@ -0,0 +1,44 @@
...
+  }
+```
+
+The above code generates a circle having dimensions of 200x200 points.
```
Comment (yangc95) [inline content/swiftui/concepts/views/terms/circle/circle.md:40]:
```suggestion
The above code generates a circle having dimensions of `200`x`200` points.
```

**Review by yangc95 — COMMENTED** [review]
@girijakar thanks for making those changes! I have a few more comments that need to be addressed prior to merge.

**Comment by girijakar** [comment]
@yangc95, implemented the latest changes. Please check :)

File: content/swiftui/concepts/views/terms/circle/circle.md [hunk 16]
```diff
@@ -0,0 +1,44 @@
...
+```
+
+`Circle` is one of the pre-built shapes. A developer can use different properties such as fill, stroke, alignment, frame, borders, and size to modify or adjust the circle according to their requirements.
+
+## Example
+
+The following example make a `Circle` that is purple:
+
+```swift
+  var body: some View{
```
Comment (yangc95) [inline content/swiftui/concepts/views/terms/circle/circle.md:33]:
```suggestion
  var body: some View {
```

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
@girijakar I made some changes prior to merge here:
[649912c](https://github.com/Codecademy/docs/pull/1138/commits/649912c4b0ead0a8fb47a0e27c540681cc1c0606) & [e13f059](https://github.com/Codecademy/docs/pull/1138/commits/e13f0590eed099a9e3a4f47b2c999a74f95048e1)

Live link: https://www.codecademy.com/resources/docs/swiftui/views/circle

cc: @SSwiniarski

**Pull request merged**